src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,26 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
```

Comment:
@jbornemann was `hasValues()` not working for the case @pgoodrich mentioned which is why this change was needed?